src/Uno.UWP/Data/Pdf/PdfPageDimensions.Android.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Uno;
+using Windows.Foundation;
+
+namespace Windows.Data.Pdf;
+
+public sealed partial class PdfPageDimensions
+{
+	[NotImplemented("__ANDROID__")]
+	public Rect ArtBox { get; }
+	[NotImplemented("__ANDROID__")]
+	public Rect BleedBox { get; }
+	[NotImplemented("__ANDROID__")]
+	public Rect CropBox { get; }
+	[NotImplemented("__ANDROID__")]
+	public Rect MediaBox { get; }
+	[NotImplemented("__ANDROID__")]
+	public Rect TrimBox { get; }
+}
```

Comment:
If this is not implemented, why it's moved out of the generated file?

Reply:
Because I don't want to throw an exception and report that it's not supported.